ipasetup.py.in

```diff
@@ -19,20 +19,21 @@ import sys
 
 
 PACKAGE_VERSION = {
-    'cryptography': 'cryptography >= 0.9',
+    'cryptography': 'cryptography >= 1.5',
     'dnspython': 'dnspython >= 1.11.1',
-    'gssapi': 'gssapi > 1.1.2',
+    'gssapi': 'gssapi >= 1.2.0',
```

Comment:
Both gssapi and cryptography has lower version in our specfile. I don't want to have inconsistent versions between RPM and pip installs